2023/01/BWbwchen/day01.rs

```diff
@@ -0,0 +1,100 @@
+use lazy_static::lazy_static;
```

Comment:
FWIW, The community now recommends [`std::sync::OnceLock`](https://doc.rust-lang.org/std/sync/struct.OnceLock.html) or [once_cell](https://crates.io/crates/once_cell).

Reply:
Cool! I will use `once_cell` instead of the `lazy_static`.